pdns/recursordist/settings/table.py

```diff
@@ -1795,6 +1795,53 @@
  ''',
     'versionadded': '4.5.0'
     },
+    {
+        'name' : 'ignore_list_file',
+        'section' : 'nod',
+        'oldname' : 'new-domain-ignore-list-file',
+        'type' : LType.String,
+        'default' : '',
+        'help' : 'File with a list of domains (and implicitly all subdomains) which will never be considered a new domain',
+        'doc' : '''
+This setting is a list of all domains (and implicitly all subdomains)
```

Comment:
This should describe the file format and reference ignore_list for the semantics